lldb/tools/lldb-dap/README.md

```diff
@@ -236,6 +236,7 @@ contain the following key/value pairs:
 | **env**                           | dictionary  |     | Environment variables to set when launching the program. The format of each environment variable string is "VAR=VALUE" for environment variables with values or just "VAR" for environment variables with no values.
 | **stopOnEntry**                   | boolean     |     | Whether to stop program immediately after launching.
 | **runInTerminal**                 | boolean     |     | Launch the program inside an integrated terminal in the IDE. Useful for debugging interactive command line programs.
+| **runInTerminal**                 | string      |     | Specifies where program should be launch: `integrated` for an integrated terminal in the IDE, `external` for external terminal window or `console` (default) for IDE debug console.
```

Comment:
I think the most common way this is implemented is a key called `console` with the values of `internalConsole`, `integratedTerminal`, or `externalTerminal`. Thats used by vscode-js-debug (https://github.com/microsoft/vscode-js-debug/blob/f8f4253879aa6cd81e5c776a4f9a619771ec2fc3/src/configuration.ts#L385C3-L385C10 its defined in code instead of the package.json) and `vscode-python` (https://github.com/microsoft/vscode-python/blob/main/package.json#L941) and `vscode-cpptools` (https://github.com/microsoft/vscode-cpptools/blob/main/Extension/package.json#L5468)

Maybe we should migrate to the new 'console' key for this value and deprecate the `runInTerminal` key. We could add a warning if we see the key and ask users to migrate.

Reply:
Also, https://code.visualstudio.com/docs/debugtest/debugging-configuration#_launchjson-attributes mentions in the common variables section the `console` key as well.

Its not strictly required, but it does match the behavior of other DAP implementations.